ui/components/breadcrumbs/flavors/base/_index.scss

```diff
@@ -21,7 +21,8 @@
       left: (($spacing-x-small / 2) * -1);
     }
 
-    > a {
+    > a,
+    span {
```

Comment:
This should probably be a `> span` to avoid propagation of styles to potential other children.